common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst

```diff
@@ -98,7 +98,7 @@ defined in a terrain database).
 .. note::
 
    The other frame types defined in the MAVLink protocol (see
-   `MAV_FRAME <https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/common.xml#L434>`__)
+   `MAV_FRAME <https://github.com/ArduPilot/mavlink/blob/master/message_definitions/v1.0/common.xml#L434>`__)
```

Comment:
This line number should be updated from 434 to 795.  These line numbers are all a bit fragile but..

Reply:
fixed